.github/workflows/deploy-cloud-node.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to DigitalOcean
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Build Docker image
+        run: make docker/cloud/build
+
+      - name: Login to DigitalOcean Docker Registry
+        run: docker login -u ${{ secrets.DO_API_KEY }} -p ${{ secrets.DO_API_KEY }} registry.digitalocean.com
+
+      - name: Push Docker image to DigitalOcean
+        run: make docker/cloud/push
+
+      - name: Deploy to Droplet
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.DO_SSH_PRIVATE_KEY }}
+        run: |
+          echo "$SSH_PRIVATE_KEY" > private_key.pem
```

Comment:
Is this a standard approach? It would be nice to avoid storing secrets on disk even if the runner is ephemeral.

Reply:
I'm not sure what the standard approach is. We might be able to read from stdin instead like this:
```
run: |
  echo "$SSH_PRIVATE_KEY" | ssh -o StrictHostKeyChecking=no -i /dev/stdin root@67.207.88.72 <<'ENDSSH'
    docker pull registry.digitalocean.com/magmo/go-nitro:latest
    docker run -it -d -p 3005:3005 -p 4005:4005 -p 5005:5005 -e SC_PK=$DO_SC_PK -e CHAIN_PK=$DO_CHAIN_PK registry.digitalocean.com/magmo/go-nitro:latest
  ENDSSH
```
I will try that instead.

Reply:
I couldn't get the stdin approach to work. I was getting permissions errors that made it seem like the ssh client would not allow the private key to be fed that way. For now, I am going to use a temporary .pem file that gets deleted as soon as the ssh session ends.